dali/python/nvidia/dali/ops.py

```diff
@@ -130,6 +130,8 @@ def _get_kwargs(schema):
                     default_value = ast.literal_eval(default_value_string)
                     type_name += ", default = `{}`".format(_default_converter(dtype, default_value))
             doc += schema.GetArgumentDox(arg)
+            if schema.ArgSupportsPerFrameInput(arg):
```

Comment:
This requires the other PR to work, doesn't it?

Reply:
Yeah, it needs to be rebased.

Reply:
Just wanted to have it ready here so that it could be read, reviewed and discussed if needed.